File: benchmarks/python/array_sum.py
import sys

def main():
    n = int(sys.argv[1]) if len(sys.argv) > 1 else 500000
    items = []
    for i in range(n):
        items.append(i % 100)
    total = 0
    for i in range(len(items)):
        total = total + items[i]
    print(total)

main()

File: benchmarks/javascript/array_sum.js
const n = Number(process.argv[2] ?? 500000);
const items = [];
for (let i = 0; i < n; i++) {
    items.push(i % 100);
}
let total = 0;
for (let i = 0; i < items.length; i++) {
    total = total + items[i];
}
console.log(total);
